keel-sql/src/main/kotlin/com/netflix/spinnaker/keel/sql/SqlDeliveryConfigRepository.kt

```diff
@@ -166,31 +188,28 @@ class SqlDeliveryConfigRepository(
         .fetchOne(DELIVERY_CONFIG.UID)
     } ?: throw NoDeliveryConfigForApplication(application)
 
-  private fun getEnvironmentUIDs(deliveryConfigUIDs: List<String>): List<String> =
+  private fun getResourceUIDs(application: String): SelectConditionStep<Record1<String>>? =
     sqlRetry.withRetry(READ) {
```

Comment:
It doesn't make sense to have a retry in here if this is returning the query itself rather than executing it

Reply:
True. Will fix.